webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -49,12 +55,21 @@ class BreadcrumbBar extends React.Component {
               onHide={() => closeSwitcher()}
               onOpen={() => this.handleOpen()}
               onResourceClick={() => closeSwitcher()}
+              onSearchChange={event =>
+                loadSwitcherResourcesByResource(resource, {
+                  search:
+                    event.target.value &&
+                    `${[resource.nameField]}~${event.target.value}`,
```

Comment:
Could you move this logic to the action? We have all necessary data there. We'll have the logic in one place and passing `search_query` from the next line won't be needed then.